File: Language_Coder/src/array/Code563.java
package array;

import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.util.Arrays;
import java.util.StringTokenizer;

public class Code563 {

	public static void main(String[] args) throws IOException {
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

		StringTokenizer st = new StringTokenizer(br.readLine());

		int[] arr = new int[10];

		for (int i = 0; i < 10; i++) {
			arr[i] = Integer.parseInt(st.nextToken());
		}

		Arrays.sort(arr);

		for (int i = 0; i < 10; i++) {
			System.out.print(arr[9 - i]);
			if (i != 9)
				System.out.print(" ");
		}
	}

}

File: Language_Coder_Cpp/Code563/Code563.cpp
// Code563.cpp

#include <iostream>
#include <algorithm>
using namespace std;

int main()
{
	int* arr = new int[10];

	for (int i = 0; i < 10; i++) {
		cin >> arr[i];
	}

	sort(arr, arr + 10);

	for (int i = 9; i >= 0; i--) {
		cout << arr[i] << " ";
	}
}